.coverage

```diff
@@ -0,0 +1 @@
lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/test_matcher.py":[1,2,3,4,24,27,34,41,50,61,70,81,90,100,115,124,133,142,151,158,167,174,28,29,30,31,35,36,37,38,42,43,44,45,46,47,51,52,53,54,55,56,57,58,62,63,64,65,66,67,71,72,73,74,75,76,77,78,82,83,84,85,86,87,92,93,94,95,96,97,102,104,106,108,110,112,116,117,118,119,120,121,125,126,127,128,129,130,134,135,136,137,138,139,143,144,145,146,147,148,152,153,154,155,159,160,161,162,163,164,168,169,170,171,175,176,177],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/matcher.py":[1,2,5,90,91,92,95,96,97,98,7,11,14,15,17,23,24,25,28,30,31,33,34,36,39,42,45,49,72,84,87,100,99,18,19,20,21,85,37,40,43,46,50,51,53,54,56,59,62,65,57,60,63,66,67,69,74,79,80,81,82,75,76,77,12,93],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/api/test_app.py":[1,3,5,8,13,9,10,14,15,17,18,20,22,24,25,28,29,34],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/api/test_app_offer.py":[1,2,3,5,6,9,10,13,16,17,18,19,20,21,22,23,24,25,26,29,30,31,32,33,34,35,36,37,38,39,43,56,66,73,74,83,90,99,100,111,112,45,46,47,48,49,50,52,58,59,60,61,62,63,68,69,70,76,77,78,79,80,85,86,87,92,93,94,95,96,102,103,104,105,106,107,108,114,115,116,117,118],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/common/test_service.py":[1,2,3,4,5,7,10,26,52,62,13,14,15,16,17,18,19,20,21,22,23,29,30,31,32,33,35,36,37,38,40,42,44,46,47,49,54,55,56,57,59,64,65,66,67,69],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/conf/test_conf.py":[1,2,4,7,17,23,9,10,11,12,13,14,19,20,25],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/db/sqlalchemy/test_api.py":[1,2,4,6,9,10,11,12,13,14,15,16,17,18,19,23,30,38,45,24,25,27,31,32,34,35,39,40,41,42,46,47,48,49,51,52],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/tests/unit/manager/test_manager.py":[1,2,3,5,6,9,10,11,12,13,23,16,17,19,20,26,27],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/manager/__init__.py":[1],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/manager/service.py":[1,2,3,4,5,7,10,11,12,15,16,26,36,38,17,18,19,20,21,22,23,24,40,27,28,29,30,31,32],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/cmd/__init__.py":[1,3],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/cmd/manager.py":[1,3,5,6,7,10,13,23,14,16,17,18,19],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/conf/opts.py":[],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/cmd/dbsync.py":[],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/cmd/api.py":[],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/api/service.py":[],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/api/controllers/__init__.py":[],"/home/lmcgann/Desktop/flocx-market/new_flocx_market/flocx_market/api/controllers/v1/__init__.py":[]}}
```

Comment:
I'm guessing this was accidentally left in as part of the commit?

Reply:
I didn't notice this file being generated. Will update the gitignore